lib/datasource/rubygems/releases.js

```diff
@@ -0,0 +1,28 @@
+const { nonEmptyArray } = require('@sindresorhus/is');
+const { getDependency } = require('./get');
+
+const RUBYGEMS_URL = 'https://rubygems.org';
+
+const getRegistries = (envSources, configSources) =>
+  Array.prototype.concat.call(envSources || configSources || RUBYGEMS_URL);
```

Comment:
Is there any time where the user would want to "forbid" Bundler from looking up the regular Rubygems service? e.g. does Bundler always query rubygems.org last no matter what, even if the user configured alterative sources in their Gemfile?

Reply:
Yep, in theory such situation can be. I add this from fallback perspective, cause [bundler requires at least one gem source](https://bundler.io/gemfile.html#gemfiles).

Reply:
So does that mean the user should always have a source anyway, and we don't need to add it manually? If they have an invalid Gemfile, I don't think we need to make it valid by adding a source ourselves. If no sources are supplied, we can just return no results.